ckcp/openshift_dev_setup.sh

```diff
@@ -281,6 +281,30 @@ patches:
   KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/ckcp-ckcp.default.compute.kubeconfig"
 }
 
+install_rekor() {
+  # Get our domain from the ingresses config of the cluster
+  echo -n "  - Domain: "
+  domain=$(kubectl get ingresses.config.openshift.io cluster -o json|jq -r ".spec.domain")
+  echo "$domain"
+
+  # Set the template input file
+  echo "  - Install: "
+  rekor_dir="$WORK_DIR/manifests/rekor"
+  mkdir -p "$(dirname "$rekor_dir")"
+  cp -rf "$SCRIPT_DIR/rekor" "$rekor_dir"
+  sed -i -e "s/\${domain}/$domain/" "$rekor_dir/$cluster_type/application.yaml"
+  kubectl apply -k "$rekor_dir/$cluster_type"
+
+  rekor_url="https://rekor-server.$domain/api/v1/log"
+  echo -n "  - Wait for api on $rekor_url: "
+  while ! curl --fail --insecure --output /dev/null --silent "$rekor_url"; do
+    echo -n .
+    sleep 3
+  done
+  echo "OK"
+  exit 0
```

Comment:
Do we really need to explicitly execute `exit 0`?

Reply:
Good catch. I forgot to remove it after testing. Fixed.